conan/api/model/remote.py

```diff
@@ -2,16 +2,24 @@
 
 
 class Remote:
-
+    """
+    The Remote class represents a remote registry of packages. It's a read-only opaque object that
+    should not be created directly, but obtained from the relevant ``RemotesAPI`` subapi methods.
+    """
     def __init__(self, name, url, verify_ssl=True, disabled=False, allowed_packages=None,
                  remote_type=None):
         self.name = name  # Read only, is the key
+        """User-facing remote name"""
         self.url = url
+        """URL endpoint of the Conan remote"""
         self.verify_ssl = verify_ssl
+        #"""Whether to verify SSL certificates"""
         self.disabled = disabled
+        #"""If this remote is disabled, it won't be taken into account for Conan operations"""
         self.allowed_packages = allowed_packages
+        #"""Which packages this remote is allowed to fetch, or None if all packages are allowed"""
         self.remote_type = remote_type
-        self.caching = {}
+        self._caching = {}
```

Comment:
Maybe this is not needed if we just dont document it

Reply:
Better to make it explicitly private, yes